ios/BT/Extensions/Other/ExposureManager+Extensions.swift

```diff
@@ -15,6 +15,12 @@ extension ExposureManager {
         }
       }
     case .detectExposuresNow:
+      guard BTSecureStorage.shared.userState.remainingDailyFileProcessingCapacity > 0 else {
+        let hoursRemaining = 24 - Date.hourDifference(from: BTSecureStorage.shared.userState.dateLastPerformedFileCapacityReset ?? Date(), to: Date())
```

Comment:
How do we feel about extracting the: `BTSecureStorage.shared.userState.dateLastPerformedFileCapacityReset ?? Date()` into a variable right on top? Also please educate me here, isn't the guard saving us from this check?

Reply:
ah, we want to get the value of `dateLastPerformedFileCapacityReset` at the time that the method is called, so I am inclined to leave it here. The code in this extension is a tad messy (it supports the debug menu, which will not be used in production)